src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -79,11 +79,15 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, MSBuild.TaskLog
         public async Task<bool> PushItemsToFeedAsync(IEnumerable<string> items, bool allowOverwrite)
         {
             Log.LogMessage(MessageImportance.Low, $"START pushing items to feed");
-            Random rnd = new Random();
 
-            try
+            if (items.Count() == 0)
```

Comment:
Prefer the following

``` c#
if (!items.Any()) 
```

It's more efficient than `.Count()` as it will iterate at most 1 element.